endpoint.go

```diff
@@ -612,6 +633,14 @@ func (ep *endpoint) setupDNS() error {
 	return ep.updateDNS(resolvConf)
 }
 
+// EndpointOptionPrimary function returns an option setter for making the
+// endpoint a primary endpoint
+func EndpointOptionPrimary() EndpointOption {
```

Comment:
Is this supposed to be set only programmatically or does it have a UI / API that goes with it ?
If it requires user control, can you also please add that in client / api packages ?